ui/apps/platform/src/Components/ReportJobsHelpAction.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Popover, TabAction } from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons';
+
+import { systemConfigPath } from 'routePaths';
+import usePermissions from 'hooks/usePermissions';
+import PopoverBodyContent from './PopoverBodyContent';
+import ExternalLink from './PatternFly/IconText/ExternalLink';
+
+type ReportJobsHelpActionProps = {
+    reportType: 'Vulnerability' | 'Scan schedule';
+};
+
+function ReportJobsHelpAction({ reportType }: ReportJobsHelpActionProps) {
+    const { hasReadWriteAccess } = usePermissions();
+    const hasAdministrationReadWriteAccess = hasReadWriteAccess('Administration');
+
+    const bodyContent = hasAdministrationReadWriteAccess ? (
+        <>
+            This function displays requested jobs from various users, including their statuses. You
+            can monitor and audit both active and past job requests. For an optimal experience, we
+            recommend configuring the{' '}
+            <ExternalLink>
+                <a href={systemConfigPath} target="_blank" rel="noreferrer">
+                    {reportType} report retention limit
+                </a>
+            </ExternalLink>{' '}
```

Comment:
1. Will do.
2. I changed this to open in a new tab from your suggestion before. I like the idea of opening a new tab to avoid disturbing the user's flow. I think it should be fine to keep.
3. Sure, we can do that.